de.lproj/Localizable.strings

```diff
@@ -458,7 +458,7 @@
 
 "COMMAND_RAW_NO_LINE" = "/raw was not given a line"; // Untranslated
 
-"COMMAND_ALIAS_DEFAULT_TITLE" = "Default Aliases:"; // Untranslated
+"COMMAND_ALIAS_DEFAULT_TITLE" = "Standard Aliases:"; // Untranslated
```

Comment:
can `// Untranslated` be removed here too, or does it still need a better translation?